src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -49,6 +49,10 @@ public AddCommand(String name,
         );
     }
 
+    public boolean isMutating () {
```

Comment:
Consider adding a @Override header comment for all methods that are overriding a method of the parent class. Makes it clear for other developers that might work on the same codebase.